website/docs/concepts2/overrides.mdx

```diff
@@ -55,4 +60,44 @@ final container = ProviderContainer(
 );
 ```
 </TabItem>
-</Tabs>
\ No newline at end of file
+</Tabs>
+
+## Inspecting overrides
```

Comment:
This is likely not worth having a decicated mention in the website. This page is quite important, yet this information is very niche in comparison.
Too much information would overwelm people.

For niche docs like this, we should be relying on dart docs.
We can document `Override.origin` for instance :)